conduit/data/datasets/utils.py

```diff
@@ -122,7 +122,9 @@ def img_to_tensor(img: Image.Image | np.ndarray) -> Tensor:
 
 def infer_al_backend() -> AudioLoadingBackend:
     """Infer which audio-loading backend to use based on the operating system."""
-    return 'soundfile' if platform.system() == 'Windows' else 'sox_io'
+    soundfile: Literal["soundfile"] = "soundfile"
+    sox: Literal["sox_io"] = "sox_io"
+    return soundfile if platform.system() == "Windows" else sox
```

Comment:
my ide can't deal with returning a str without being of literal type

Reply:
That's a bit shit